scripts/astrokat-targets.py

```diff
@@ -491,12 +507,16 @@ def table_line(datetime,
     [rise_time,
      set_time] = observatory.target_rise_and_set_times(target.body,
                                                        lst=lst)
-    if not lst:
+
+    if type(rise_time) is ephem.Angle:
```

Comment:
When testing for the type, you should instead use [isinstance(tmpDict[key], list)](https://docs.python.org/3.4/library/functions.html#isinstance) when testing the type of something, apparently, it is a more Pythonic way of checking the type :smile: 
```suggestion
    if isinstance(ephem.Angle, rise_time):
```